platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```diff
@@ -145,34 +158,42 @@ public SecureCxfClientFactory(String endpointUrl, Class<T> interfaceClass, List<
      */
     public SecureCxfClientFactory(String endpointUrl, Class<T> interfaceClass, List<?> providers,
             Interceptor<? extends Message> interceptor, boolean disableCnCheck,
-            Integer connectionTimeout, Integer receiveTimeout, String username, String password) {
+            boolean allowRedirects, Integer connectionTimeout, Integer receiveTimeout,
+            String username, String password) {
 
-        this(endpointUrl, interfaceClass, providers, interceptor, disableCnCheck, connectionTimeout,
-                receiveTimeout);
+        this(endpointUrl, interfaceClass, providers, interceptor, disableCnCheck, allowRedirects,
+                connectionTimeout, receiveTimeout);
 
         this.clientFactory.setPassword(password);
```

Comment:
I debated this as well, but technically you could have an empty password, and the javadoc for the JAXRSClientFactoryBean don't mention any restrictions.

I could still be convinced to change it if we have a strong opinion.